src/Events/Editor/Full_Site/Templates.php

```diff
@@ -26,14 +26,23 @@ class Templates {
 	 */
 	public static $archive_slug = 'archive-' . Tribe__Events__Main::POSTTYPE;
 
+	/**
+	 * The single slug.
+	 *
+	 * @since TBD
+	 *
+	 * @var string The single slug.
+	 */
+	public static $single_slug = 'single-' . Tribe__Events__Main::POSTTYPE;
+
 	/**
 	 * Modify the available Templates so that people can edit the template.
 	 *
 	 * @since 5.14.2
 	 *
 	 * @param WP_Block_Template[] $query_result Array of found block templates.
-	 * @param array  $query {
-	 *     Optional. Arguments to retrieve templates.
+	 * @param array               $query        {
```

Comment:
```suggestion
	 * @param array               $query        
```
I know this wasn't you, but is that curly brace intended? Looks off to me.

Reply:
I think that's intentional. Additional details provided in the curly braces give a more in-depth description of the structure and intent of this $query parameter.